spark-wrapper/spark-3.0/src/main/scala/org/apache/spark/sql/extensions/rules.scala

```diff
@@ -40,60 +38,52 @@ class TiResolutionRuleFactory(getOrCreateTiContext: SparkSession => TiContext)
 case class TiResolutionRule(getOrCreateTiContext: SparkSession => TiContext)(
```

Comment:
would u please add a new `TiResolutionRuleV2` instead of modify `TiResolutionRule`?